Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2759,6 +2759,20 @@ ExposeDefaultSpeakerAsSpecificDeviceEnabled:
     WebCore:
       default: true
 
+ExposeRollAngleAsTwistEnabled:
+  type: bool
+  status: stable
```

Comment:
nit: Stable doesn't seem like the most appropriate category, I would've gone with Internal or Testable?